### PR TITLE
[Ready for Review] Use of AngularJS markup in URL-valued attribute

### PR DESCRIPTION
**Pull request by MatheusRV**
## What does this PR (Pull Request) do?
Using AngularJS markup in an HTML attribute that references a URL (such as 'href' or 'src') may cause the browser to send a request with an invalid URL.

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html [hunk 1]
```diff
@@ -356,7 +356,7 @@ <h6>Note:</h6>
                             Enables/disables the Multi-Site Origin feature for this Delivery Service.
                             <br>
                             <br>
-                            <a href="https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#multi-site-origin" target="_blank">See Multi-Site Origin.</a>
+                            <a ng-href="https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#multi-site-origin" target="_blank">See Multi-Site Origin.</a>
```
Comment (ocket8888) [inline traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html]: This - and several others at first glance - does not appear to be using any AngularJS string interpolation (which is I think what you mean by "markup"?) so this is an unnecessary change.
